.gitignore

```diff
@@ -1,4 +1,5 @@
 build
 dist
+src
```

Comment:
When pip checks out editable projects and you are running in a virtualenv, pip places them by default in `<venv path>/src`. For global installs, the default is `<current dir>/src`.

You can change pip behaviours by passing the `--src` option:
`pip install --src <dir> -r dev_requirements.txt`

And you can configure your local copy of the repo to ignore some files by doing, e.g.:

```
echo "src/" >> .git/info/exclude
```